### ISSUE TITLE
Running npx gsn start results in an error

### ISSUE THREAD

**Issue by pmwiseman** [opengsn/gsn#644]
Hi, thanks for releasing v2.2.1.  I'm trying to upgrade, but for some reason when I run `npx gsn start` I get the following error:

```
npx: installed 1 in 0.83s
command not found: gsn
```

Do I still have to keep the monorepo as a dependency?  Thanks.

**Comment by pmwiseman** [comment]
Hi, I'm back with an update. I managed to get it working. Here's what I found:
- It seems like this [file](https://github.com/opengsn/gsn/blob/66328e9bb7286d8559d798d873df6f3a79ec1970/packages/cli/src/commands/gsn-start.ts#L1) is commented out so the `gsn start` command won't work.  I uncommented it and updated the code locally to resolve the issue.
- In the end I could not get `npx gsn start` to work, but I can use `node node_modules/@opengsn/cli/dist/commands/gsn.js` instead.
- I also had an additional issue with an `invalid address` error that was throwing [here](https://github.com/ChainSafe/web3.js/blob/1.x/packages/web3-core-helpers/src/formatters.js#L471). For some reason installing `ethereumjs-wallet` at version `0.6.5` resolved the issue. I'm not sure why this works.

Let me know if I should close this issue or make a PR to fix the commented out code in the `gsn-start` file. Thank you!
